src/renderers/webvr/WebXRManager.js

```diff
@@ -121,16 +144,19 @@ function WebXRManager( renderer ) {
 			session.addEventListener( 'end', onSessionEnd );
 
 			session.baseLayer = new XRWebGLLayer( session, gl, { framebufferScaleFactor: framebufferScaleFactor } );
```

Comment:
Add new support for updateRenderState() to get the session's `baseLayer`.

	if ( session.updateRenderState !== undefined ) {

		session.updateRenderState( {  baseLayer:  new XRWebGLLayer( session, gl, {  framebufferScaleFactor: framebufferScaleFactor } ) } );

	} else {

		session.baseLayer =  new XRWebGLLayer( session, gl, {  framebufferScaleFactor: framebufferScaleFactor } );

	}

Reply:
https://github.com/mrdoob/three.js/pull/15604#discussion_r257491150